src/coreclr/src/jit/lsrabuild.cpp

```diff
@@ -2266,7 +2263,7 @@ void LinearScan::buildIntervals()
                     {
                         JITDUMP("  Marking RP #%d of V%02u as spillAfter\n", interval->recentRefPosition->rpNum,
                                 interval->varNum);
-                        interval->recentRefPosition->spillAfter;
+                        interval->recentRefPosition->spillAfter = true;
```

Comment:
Could it produce any diffs?

Reply:
That is a good question. May be I will validate that and post it here.

Reply:
Good catch, @sandreenko!

Reply:
Verified crossgen and PMI on framework libraries and I don't see any diffs.